packages/web/example/SendCIS2UpdateOperator.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <script src="../lib/concordium.min.js"></script>
+    <script type="module">
+        const client = concordiumSDK.createConcordiumClient("https://grpc.testnet.concordium.com", 20000);
+
+        const send = async function () {
+            // The better approach here though, would be to install "@concordium/browser-wallet-api-helpers" and use the supplied `detectConcordiumProvider` function.
+            const walletApi = window.concordium;
```

Comment:
Thought about this as well, however it is just an example and I think there is a case for being pragmatic here. The example is not very interesting without actually processing the transaction in a way most users will process the transaction (in my opinion), and the case of sending a transaction using a private key is already covered in a nodeJS example, and is not a very good use case for the browser environment anyway.

Reply:
If you disagree, how would you examplify this in a good and useful way?

Reply:
I guess you could just invoke it :shrug: 
Otherwise I think a reasonable compromise could be that you add these examples to the readme, and in there note that they require the browser wallet to function.